tests/utils/notebooks_test_utils.py

```diff
@@ -82,3 +83,28 @@ def _ex_dataset_download(dataset: str, save_dir: str, cache_dir: Union[str, None
     ex_dataset.download_example_dataset()
 
     ex_dataset.move_example_dataset(move_dir=save_dir)
+
+
+def get_storage(method):
```

Comment:
Where does this get used? Or is it just for future use if something comes up again?

Reply:
It isn't used anywhere right now, but I did use it to investigate how storage space changes over time through the example dataset tests and the notebook tests.

It should only be used for that purpose, and we don't need to decorate any function with it currently.